buildSrc/src/main/java/Dependencies.kt

```diff
@@ -20,10 +21,11 @@ object Versions {
 }
 
 object Deps {
-    val app_compat_x = "androidx.appcompat:appcompat:${Versions.android_x}"
-    val material_x = "com.google.android.material:material:${Versions.android_x}"
-    val core_ktx = "androidx.core:core-ktx:${Versions.android_x}"
-    val constraintlayout = "androidx.constraintlayout:constraintlayout:${Versions.android_x}"
+    val app_compat_x = "androidx.appcompat:appcompat:1.1.0"
+    val material_x = "com.google.android.material:material:1.1.0"
+    val core_ktx = "androidx.core:core-ktx:1.2.0"
+    val constraintlayout = "androidx.constraintlayout:constraintlayout:1.1.3"
+    val recyclerView = "androidx.recyclerview:recyclerview:1.1.0"
```

Comment:
Since androidx artifacts are independently versioned (unlike old support library stuff), I removed `Versions.android_x` and just inlined these. Open to keeping the version numbers extracted as separate constants if we prefer that stylistically, but they shouldn't be sharing the same constant.